common_benchmark_suite/openxla/benchmark/comparative_suite/tf/model_definitions.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from openxla.benchmark import def_types, unique_ids
+from openxla.benchmark.comparative_suite import utils
+
+PARENT_GCS_DIR = "https://storage.googleapis.com/iree-model-artifacts/tensorflow/tf_models_2.12.0_1685428719/"
+
+T5_LARGE_FP32_TF_IMPL = def_types.ModelImplementation(
```

Comment:
I think technically this can be called `T5_TF_IMPL` as it is configurable with different datatypes and sizes.

Reply:
Thanks, updated.